components/Core/Carousel/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react'
+import { FlatButton, VerticalLayout, HorizontalLayout } from '../'
+import styles from './Carousel.module.css'
+
+interface CarouselProps {
+  children: React.ReactNode[]
+}
+
+
+const Carousel: FC<CarouselProps> = ({ children }) => {
+  const [ currentIndex, setCurrentIndex ] = useState<number>(0)
+  const prevItem = () => {
+    if (children.length < 2) {
```

Comment:
I'm not too sure if I understand why number 2 is present here. Is it like for disabling carousel if there is not enough items? I would be bit more explicit maybe in a way that you would have 
``const controlsDisabled = children.length < 2`` and use that to maybe hide useless buttons. Not sure if I undrestand this correctly tho

Reply:
There is always one item in this carousel, at least currently. IIRC, that is why I opted for this.